### DISCUSSION TITLE
list navigation using vim-like commands

### DISCUSSION THREAD

**Discussion by bkemmer** [alefragnani/vscode-bookmarks#751] [Q&A] [answered]
Hi there,
First, thank you for the awesome extension, it is helping me keeping things organized here :-)

Second, I am trying to move to vim biddings and for me it will be useful to keep using the navigation that we have in the autocomplete suggestions, `alt+j` and `alt+k` to move between the bookmaks in `bookmarks.list`. 

```json
      {
        "key": "alt+j",
        "command": "bookmarks.<command to go down>", # current arrow down
        "when": "bookmark.list" # is this the right context?
      },
      {
        "key": "alt+k",
        "command": "bookmarks.<command to go up>", # current arrow up
        "when": "bookmark.list" # is this the right context?
      },
```

Also, it would be interesting deleting a bookmark inside the list, but lets leave this to later.

<img width="843" alt="Screenshot 2024-09-19 at 10 32 23" src="https://github.com/user-attachments/assets/b35a71aa-e863-4daa-a0bc-4e324d013bf7">

**Comment by alefragnani** [comment, marked as answer]
Hi @bkemmer ,

There is no _context_ today so, this is not possible yet. But I think something like `inBookmarkList` context could added.

Thanks for your suggestion

**Reply by alefragnani** [reply]
Hi @bkemmer ,

You could try the _native_ `inQuickOpen` clause instead, as it works for any quick pick list. Something like this:

```json
      "command": "workbench.action.quickOpenSelectNext",
      "when": "inQuickOpen"
```

Hope this helps
